beta-src/src/components/ui/WDInfoDisplay.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  TableContainer,
+} from "@mui/material";
+
+/**
+ * game setting datas which would be passed to the component by parent component/ context/redux store
+ */
+
+interface WDInfoDisplayProps {
+  title: string;
+  gameTime: string;
+  phase: string;
+  season: string;
+  year: string;
+  gameType: string;
+  playType: string;
+  rank: string;
```

Comment:
I believe there are only `ranked` and `unranked` games so let's make this an enum as well but you need to double check this with the PO / in game